src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +130,14 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts the names of all persons in list by alphabetical order.
+     *
+     * @param comparator
+     */
+
+    public void sort(Comparator comparator) { Collections.sort(internalList, comparator); }
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. brace style problem.

Futhermore it would be better to change the argument `Comparator` to `NameComparator` to add more clarity to the method. Else other developers may accidentally use any other comparator to sort.